cypress/e2e/login/logoutTest.cy.js

```diff
@@ -0,0 +1,124 @@
+describe('Login/logout functionality and authentication', () => {
+    // Checks if a user can successfully log in
+    it('Login functionality', () => {
+      cy.visit('/l');
+      cy.wait(3000);
+  
+      // Open the login modal
+      cy.get("button[data-auth='login']")
+        .filter(':visible')
+        .first()
+        .click({ force: true });
+      cy.wait(3000);
+  
+      // Enter valid credentials
+      cy.get("input[type='email']")
+        .filter(':visible')
+        .first()
+        .should('exist')
+        .type('test@noroff.no');
+      cy.get("input[type='password']")
+        .filter(':visible')
+        .first()
+        .should('exist')
+        .type('Password1234');
+      cy.get('.btn-success').filter(':visible').first().click({ force: true });
+      cy.wait(3000);
+  
+      // Assert the URL to include 'view=profile'
+      cy.url({ timeout: 10000 }).should('include', 'view=profile');
```

Comment:
10 seconds seems a bit extreme, 3 should be fine unless you are on a very slow network